src/Microsoft.AspNetCore.DataProtection/RegistryPolicyResolver.cs

```diff
@@ -14,10 +14,15 @@
 
 namespace Microsoft.AspNetCore.DataProtection
 {
+    internal interface IRegistryPolicyResolver
```

Comment:
Can you add a comment explaining why we have an internal only interface with only one implementation? I can easily imagine some future dev removing this because we don't have tests for the scenarios that this fixes.